reporter/html.js

```diff
@@ -782,9 +782,11 @@ export function escapeText( s ) {
 					escapeText( actual ) + "</pre></td></tr>";
 
 				if ( typeof details.actual === "number" && typeof details.expected === "number" ) {
-					showDiff = true;
-					diff = ( details.actual - details.expected );
-					diff = ( diff > 0 ? "+" : "" ) + diff;
+					if ( !isNaN( details.actual ) && !isNaN( details.expected ) ) {
```

Comment:
As expected/actual are displayed anyway, I don't think it matters what the diff says when a value is NaN. As it doesn't really add any information we might as well not show it.